src/System.Security.Cryptography.Algorithms/ref/4.1/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "System.Runtime": "4.0.0",
+    "System.IO": "4.0.0",
+    "System.Security.Cryptography.Primitives": "4.0.0-rc3-24026-00"
```

Comment:
rc3 is correct for master.  System.Security.Cryptography.Primitives doesn't have any stable versions, so this looks correct to me.